tests/test_service.py

```diff
@@ -81,17 +83,150 @@ def test_index(self):
+                                       }]
+                                   },
+                             content_type='application/json')
+
+        self.assertEqual(resp.status_code, status.HTTP_201_CREATED, 'Could not create order')
+
+    def test_create_orders_wrong_content_type(self):
+        """ create an order """
+        resp = self.app.post('/orders',
+                             json={"customer_id": 123,
+                                   "order_items": [
+                                       {
+                                           "quantity": 5,
+                                           "price": 500,
+                                           "status": "PLACED"
+                                       }]
+                                   },
+                             content_type='application/xml')
+
+        self.assertEqual(resp.status_code, status.HTTP_415_UNSUPPORTED_MEDIA_TYPE, 'Could not create order')
+
+    def test_create_orders_customer_id_missing(self):
+        """ create an order missing customer_id """
+        resp = self.app.post('/orders',
+                             json={"customer_id": None,
+                                   "order_items": [
+                                       {
+                                           "quantity": 5,
+                                           "price": 500,
+                                           "status": "PLACED"
+                                       }]
+                                   },
+                             content_type='application/json')
+
+        self.assertEqual(resp.status_code, status.HTTP_400_BAD_REQUEST, 'Could not identify bad request')
+
+    def test_create_orders_customer_id_wrong_type(self):
+        """ create an order with invalid customer_id """
+        resp = self.app.post('/orders',
+                             json={"customer_id": "abc",
+                                   "order_items": [
+                                       {
+                                           "quantity": 5,
+                                           "price": 500,
+                                           "status": "PLACED"
+                                       }]
+                                   },
+                             content_type='application/json')
+        self.assertEqual(resp.status_code, status.HTTP_400_BAD_REQUEST, 'Could not identify bad request')
+
+    def test_create_orders_customer_id_order_items_missing(self):
+        """ create an order missing order_items """
+        resp = self.app.post('/orders',
+                             json={"customer_id": 123,
+                                   "order_items": None
+                                   },
+                             content_type='application/json')
+
+        self.assertEqual(resp.status_code, status.HTTP_400_BAD_REQUEST, 'Could not identify bad request')
+
+    def test_create_orders_customer_id_order_items_missing(self):
```

Comment:
The function defined in line 146 has the same name. Is this on purpose? Suggest renaming to "test_create_orders_customer_id_order_items_missing_product".